core/backend/src/IpcHost.ts

```diff
@@ -91,7 +91,7 @@ export class IpcHost {
   }
 
   public static async startup(opt?: { ipcHost?: IpcHostOptions, iModelHost?: IModelHostConfiguration }): Promise<void> {
-    this._ipc = opt?.ipcHost?.socket;
+    this._ipc = opt?.ipcHost?.socket || new IpcWebSocketBackend();

```

Comment:
some of our tests create a IpcHost without a frontend (which is why I allowed undefined). Will this be ok?

BTW, i think ?? is better than || now.

Reply:
It seems to me this would be better done in MobileHost.startup (set ipcHost.socket there)

Reply:
I will move it.